mayavi/core/lut_manager.py

```diff
@@ -429,6 +429,8 @@ def _use_default_range_changed(self, value):
         self._default_data_range_changed(self.default_data_range)
 
     def _data_range_changed(self, value):
+        # should be guaranteed by callers, otherwise VTK will print an error
+        assert value[0] <= value[1]
```

Comment:
It's indicative that some caller has done something bad. So really it should be fixed (as was done here). But in principle something like `value = np.array([np.mean(value)] * 2)` could be used here (probably better to make them equal rather than swap).